js/shopping-list-item.js

```diff
@@ -0,0 +1,40 @@
+class Item {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.isDone = false;
+  }
+  check() {
+    this.isDone = true;
+  }
+  uncheck() {
+    this.isDone = false;
+  }
+  render(num) {
+    let checkBox;
```

Comment:
This entire function could be re-written to utilize string templates rather than using the `+` operator to concat strings together.